.github/workflows/discord-bot-deployment.yml

```diff
@@ -17,3 +17,30 @@ jobs:
         run: cd discord-bot && npm ci
       - name: Deploy Serverless Bot on AWS
         run: cd discord-bot && npm run deploy
+        env:
+          # Discord Bot Token, Guild ID and Client ID
+          TOKEN: ${{ secrets.TOKEN }}
+          PUBLIC_KEY: ${{ secrets.PUBLIC_KEY }}
+          CLIENT_ID: ${{ secrets.CLIENT_ID }}
+          GUILD_ID: ${{ secrets.GUILD_ID }}
+
+          # Faucet Contract Address and Minter Private Key
+          CONTRACT_ADDRESS: '0x872f3534a525023F9E42Ad69b788B44fc958D5aa'
```

Comment:
Curious to hear what is the reason for keeping the contract address outside of the .env? It's easier to replace the value there than make changes to the workflow each time, and likely less error-prone.

Reply:
Only a admin of this repo can change the repo actions secrets, while anyone from the organization can approve a PR triggering the bot to be redeploy, that was the reason.
I'll change it for a secret :)